flocker/node/agents/blockdevice.py

```diff
@@ -813,12 +830,12 @@ def destroy_volume(blockdevice_id):
         :return: A ``Deferred`` that fires when the volume has been destroyed.
         """
 
-    def attach_volume(blockdevice_id, host):
+    def attach_volume(blockdevice_id, storage_backend_id):
```

Comment:
While working on EBS attach_volume, i was wondering about usecase where attach_volume is run from a compute instance that is different from the compute instance where we want to attach the block device. Since this code is part of node agent that runs on a compute instance, is the following scenario ever possible?
- 1 cluster (say EC2) with 2 instances
- Will node agent on instance1 every want to run:
attach_volume(block device, instance2) 
?

i thought node agents only execute actions for local node. Please correct me if that is wrong? If not, does attach_volume need storage_backend_id parameter? Thanks!